pkg/odo/cli/url/delete.go

```diff
@@ -57,15 +61,20 @@ func (o *URLDeleteOptions) Validate() (err error) {
 
 // Run contains the logic for the odo url delete command
 func (o *URLDeleteOptions) Run() (err error) {
+
```

Comment:
It seems from https://docs.google.com/document/d/1RKva9lvIR6dLmDm2tlqmp8V4nFjlrXdulOQ4FSmLkr8/edit?ts=5c715ffd that url delete will now take optional `--push` flag only on which the url is deleted otherwise the doc seems to suggest `odo push` needs to handle url deletions as well so if I understand correctly push above is not deleting urls?
But please confirm once that url deletions need to happen as part of push? bcoz if user creates urls via non-odo tool, odo config will not have it and hence `odo push` will destroy that url

Reply:
if user creates url via non-odo tools, we dont care about it because it will not have odo specific labels